app/presenters/teaching_events/event_presenter.rb

```diff
@@ -86,9 +86,9 @@ def quote
 
     def image
       case event_type
-      when "Train to Teach event"
+      when "Get Into Teaching event"
         {
-          path: "media/images/content/event-signup/birmingham-event-1.jpg",
+          path: "media/images/content/event-signup/event-regional.jpg",
           alt: "A bustling Train to Teach event taking place in a church, busy with stalls and visitors",
```

Comment:
```suggestion
          alt: "A busy Get Into Teaching event with people having one-on-one conversations with expert advisers and teachers",
```

@gazcarr Are you happy with the new image - I think it looks much better?

@ethax-ross Once Gary confirms, please could you update the alt text as suggested